client/my-sites/woocommerce/main.jsx

```diff
@@ -2,19 +2,33 @@
  * External dependencies
  */
 import React from 'react';
+import { useSelector } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import RequiredPluginsInstallView from './dashboard/required-plugins-install-view';
 import WooCommerceColophon from './woocommerce-colophon';
 import Main from 'calypso/components/main';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { isLoaded as arePluginsLoaded } from 'calypso/state/plugins/installed/selectors';
+import QueryJetpackPlugins from 'calypso/components/data/query-jetpack-plugins';
 
 function WooCommerce() {
+	const siteId = useSelector( getSelectedSiteId );
+	const areInstalledPluginsLoadedIntoState = useSelector( ( state ) =>
```

Comment:
This relies on `siteId` being present. Should it be moved below the `if ( ! siteId )` check?

Reply:
I had originally had the code that way, but got a linter warning/error that hooks should always be above any conditionals, since they should always all run. Perhaps I should put a check for `siteId` into the anonymous function just to make sure that I'm not relying on `arePluginsLoaded handling `siteId` being null/undefined in a certain way. But, given that the result isn't used unless `siteId` is defined, maybe this isn't really an issue.

Reply:
Makes sense! Let's keep it as-is then.